solvis/geometry.py

```diff
@@ -133,15 +189,20 @@ def bearing(point_a: Point, point_b: Point) -> float:
 
 
 strike = bearing  # alias for bearing function
+"""An alias for the [`bearing`][solvis.geometry.bearing] function."""
 
 
 def refine_dip_direction(point_a: Point, point_b: Point, original_direction: float) -> float:
     """
     Compute a dip direction that fits the orientation established by the original_direction.
```

Comment:
@chrisdicaprio can you provide some words of context here (i.e. why is this useful?)  I think it's because scientists / libraries don't obey the same dip-direction convention ?? and this helps us sort the mess out :)

Reply:
to clarify what this function does:
- change `original_direction` to `original_dip_direction`
- Change the first line of the doc string to be:
```
Compute the dip direction given two points along strike. Rather than obey the right hand rule for dip direction, this function will return a dip oriented in the same direction as the original_dip_direction. This is useful when re-calculating the dip direction of OpenSHA fault subsections from the fault surface trace, but keeping the orientation (i.e. resolving the 180 degree ambiguity when not obeying the right hand rule).
```

Also, there is a possible failure mode of this function. It's very unlikely, but to protect against it I would suggest:
```
def resolve_azimuth(az: float) -> float:
    """
    Resolve an azimuth angle to be between -360 and 360

    Parameters:
        az: the azimuth angle in decimal degrees

    Returns:
        The resolved azimuth angle in decimal degrees
    """
    return math.copysign(az % 360.0, az)
```

And in `refine_dip_direction()`
```
original_direction = resolve_azimuth(original_direction)
```

Reply:
awesome - thanks @chrisdicaprio !! @DrCuriosity canl you integrate this text pls?

Reply:
Have dropped these in now.